bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -96,10 +96,20 @@ spec:
         - mountPath: /etc/openvswitch
           name: host-config-openvswitch
         resources:
+          limits:
+            cpu: 200m
+            memory: 400Mi
           requests:
-            cpu: 100m
-            memory: 300Mi
+            cpu: 200m
+            memory: 400Mi
         terminationMessagePolicy: FallbackToLogsOnError
+        livenessProbe:
+          exec:
+            command:
+            - cat
```

Comment:
This should just be `/usr/share/openvswitch/scripts/ovs-ctl status`. Then you can get replace the loop in the pod's command with just `sleep 10000`.